src/components/AppForm.vue

```diff
@@ -25,7 +25,7 @@
     </fieldset>
 
     <fieldset>
-      <label for="columngap">Column Gap</label>
+      <label for="columngap">Column Gap (in px)</label>
```

Comment:
This needs a span and class that's got the opacity lower to convey heirarchy. Probably either `opacity: 0.85` or `opacity: 0.9`

Reply:
I've added extra spans with opacity. Sorry for late response. I hope you'll like the final effect.